src/SGB_Command_Sound.md

```diff
@@ -44,16 +44,21 @@ Used to transfer sound code or data to SNES Audio Processing Unit memory
  1-F   Not used (zero)
 ```
 
-The sound code/data is sent by VRAM-Transfer (4 KBytes).
+The sound code/data is sent by [VRAM transfer](<#VRAM Transfers>) as a contiguous list of "packets".
```

Comment:
Should "data" be enough here instead of the potentially confusing "code and/or data"?

Reply:
Well, most applications of `SOU_TRN` will only upload custom data for Kankichi-kun, but it's possible to upload a custom sound driver as well. Hence "code and/or data".

Reply:
@ISSOtm okay, that makes sense. Thanks for the clarification.